girls.css

```diff
@@ -540,3 +540,7 @@ h1:after {
 .is-loading h1:after {
 	width: 0;
 }
+
+.col{
```

Comment:
this is a bootstrap class, rename this class. It's not a good way to change framework classes.